src/main/kotlin/no/nav/syfo/personoppgave/PersonOppgaveQueries.kt

```diff
@@ -181,24 +211,26 @@ fun Connection.createPersonOppgave( // TODO: send in oppgave instead of dialogmo
 fun Connection.createPersonOppgave(
     melding: Melding,
     personOppgaveType: PersonOppgaveType,
-) {
+): UUID {
     val now = Timestamp.from(Instant.now())
+    val uuid = UUID.randomUUID()
 
     val personIdList = prepareStatement(queryCreatePersonOppgave).use {
-        it.setString(1, UUID.randomUUID().toString())
+        it.setString(1, uuid.toString())
         it.setString(2, melding.referanseUuid.toString())
         it.setString(3, melding.personIdent.value)
         it.setString(4, "")
         it.setString(5, personOppgaveType.name)
         it.setTimestamp(6, now)
-        it.setTimestamp(7, Timestamp.from(melding.tidspunkt.toInstant()))
-        it.setBoolean(8, true)
+        it.setTimestamp(7, now)
+        it.setBoolean(8, false)
```

Comment:
Publish-feltet er bare ment til bruk for cronjob-logikk, så meldingsoppgaver skal ikke lenger bry seg om dette.